### DISCUSSION TITLE
Zeus not connecting with core lightning

### DISCUSSION THREAD

**Discussion by n-e-t-d-i-v-e-r** [ZeusLN/zeus#1435] [Q&A]
hi there
trying to connect zeus wallet with my core lightning node.

for doing this I used this manual: https://raspibolt.org/guide/bonus/lightning/cln.html#connect-zeus

config is:
HiddenServiceDir /var/lib/tor/hidden_service_cln_rest/
HiddenServiceVersion 3
HiddenServicePort 9090 127.0.0.1:9090

has anybody an idea?

![IMG_9605](https://user-images.githubusercontent.com/87816768/232209468-e01246c1-a6c0-4012-8bfa-939bfe468ac7.jpg)

**Comment by n-e-t-d-i-v-e-r** [comment]
maybe this is also an useful information for fixing my zeus-connection-issue:


# network
proxy=127.0.0.1:9050
bind-addr=0.0.0.0:9736
addr=statictor:127.0.0.1:9051/torport=9736
always-use-proxy=true

# cln-rest-plugin
plugin=/data/lightningd-plugins-available/c-lightning-REST-0.10.1/clrest.js
rest-port=3092
rest-docport=4092
rest-protocol=http

looks for me as I’ve got an port cofig issue. but don’t know where and how to fix it.

**Comment by sha-265** [comment]
You should create `HiddenServicePort` with the same port as the `rest-port`.

Something like this:
```
HiddenServicePort 3092 127.0.0.1:3092
```
